src/launch.py

```diff
@@ -30,28 +29,32 @@ def typer_interface(config_file: str):
     @staticmethod
     def launch(config):
 
+
         text_prompt_manager = TextPromptManager()
 
         start_time = time.time()
+        collection_name = "Collection"
         bpy.ops.wm.open_mainfile(filepath="src/blender_assets/CropAssets.blend")
-        collection = "Collection"
+        for obj in bpy.data.objects:
+            obj.location = (1000,1000,1000)
         # Set the unit system to metric
         bpy.context.scene.unit_settings.system = "METRIC"
         bpy.context.scene.unit_settings.scale_length = 1.0  # Set the scale to 1.0 for metric units
-        cropcon = CropController(config, collection)
-        scenerender = SceneRenderer(config, collection)
+        cropcon = CropController(config, collection_name)
+        scenerender = SceneRenderer(config, collection_name)
         cropcon.setup_crops()
         scenerender.setup_render()
-        planting_date = config["planting_date"]
-        lat = config["latitude"]
-        lon = config["longitude"]
-        barley_type = config["barley_type"]
-        api_key = os.environ["WEATHER_API"]
-        weather_controller = WeatherController(api_key)
-        weather_data = weather_controller.get_merged_weather_data(barley_type, planting_date, lat, lon)
+        # planting_date = config["planting_date"]
+        # lat = config["latitude"]
+        # lon = config["longitude"]
+        # barley_type = config["barley_type"]
```

Comment:
Can this commented out code be removed if weather controller is being defined else where